core/src/dird/reload.cc

```diff
@@ -183,20 +182,35 @@ bool CheckResources()
     }
   }
 
-  if (OK) {
-    CloseMsg(nullptr);              /* close temp message handler */
-    InitMsg(nullptr, me->messages); /* open daemon message handler */
-    if (me->secure_erase_cmdline) {
-      SetSecureEraseCmdline(me->secure_erase_cmdline);
-    }
-    if (me->log_timestamp_format) {
-      SetLogTimestampFormat(me->log_timestamp_format);
+  ConsoleResource* console;
+  BStringList consoles_with_auth_problems;
+  foreach_res (console, R_CONSOLE) {
+    if (console->use_pam_authentication_
+        && (console->user_acl.ACL_lists[0] || console->user_acl.profiles)) {
```

Comment:
if I understood it correctly, then `ACL_lists[0]` is only checking for Job_ACL, but not for the others like Catalog_ACL or Command_ACL.